src/Config/ValueProvider/SourceDirsProvider.php

```diff
@@ -60,6 +58,11 @@ public function get(InputInterface $input, OutputInterface $output, array $dirsI
 
         $choices = array_merge(['.'], array_values($dirsInCurrentDir));
 
+        // if a default value is not any of current choices, include it as the first of allowed values
+        if (!in_array($defaultValues, $choices)) {
```

Comment:
`$defaultValues` can contain more than just one guessed folders, e.g. `src/A,src/B`?

In this case, I guess it's not correct to insert this comma-separated value as a choice, because we will have

```
[ ] src/A,src/B
[ ] src
[ ] otherFolder
```

I think you should work here with `$guessedSourceDirs` array instead of string `$defaultValues`

Could you please add a test for this case?

Reply:
Yes, I should have used an array. Fixed.